pkg/crdconversion/crdconversion_test.go

```diff
@@ -96,14 +95,13 @@ func TestUpdateCrdConfiguration(t *testing.T) {
 func TestNewConversionWebhook(t *testing.T) {
 	assert := tassert.New(t)
 	crdClient := fake.NewSimpleClientset()
-	kubeClient := k8sfake.NewSimpleClientset()
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
 	fakeCertManager := tresorFake.NewFake(1 * time.Hour)
 	osmNamespace := "-osm-namespace-"
 	enablesReconciler := false
 
-	actualErr := NewConversionWebhook(context.Background(), kubeClient, crdClient.ApiextensionsV1(), fakeCertManager, osmNamespace, enablesReconciler)
+	actualErr := NewConversionWebhook(context.Background(), crdClient.ApiextensionsV1(), fakeCertManager, osmNamespace, enablesReconciler)
```

Comment:
are these changes supposed to be here as well? if not, can you rebase off upstream/main?

Reply:
kubeclient isn't used in this function, noticed it when I was cleaning up. Would you prefer that as a separate PR?

Reply:
gotcha, no it's fine will resolve